src/main/java/core/basesyntax/service/activitieswithfruitservice/ActivitiesWithFruitService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service.activitieswithfruitservice;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface ActivitiesWithFruitService {
+    void performTransaction(FruitTransaction fruitTransaction);
```

Comment:
```suggestion
public interface TransactionHandler {
    void performTransaction(FruitTransaction fruitTransaction);
```

Reply:
not fixed. you can't just ignore Mentor's suggestions. if you have any questions or need some help - they should be asked in the platform's chats